src/engraving/internal/engravingconfiguration.cpp

```diff
@@ -68,6 +69,7 @@ void EngravingConfiguration::init()
         Settings::Key key("engraving", "engraving/colors/voice" + std::to_string(voice + 1));
 
         settings()->setDefaultValue(key, Val(defaultVoiceColors[voice].toQColor()));
+        settings()->setDescription(key, qtrc("engraving", "Voice").toStdString() + " " + std::to_string(voice + 1) + " " + qtrc("engraving", "color").toStdString());
```

Comment:
The preferred way to do this is:
```suggestion
        settings()->setDescription(key, qtrc("engraving", "Voice %1 color").arg(voice + 1).toStdString());
```